driver-core/src/main/java/com/datastax/driver/core/NettyOptions.java

```diff
@@ -183,22 +183,17 @@ public void afterChannelInitialized(SocketChannel channel) throws Exception {
      * <p>
      * This gives the implementor a chance to close the {@link EventLoopGroup} properly, if required.
      * <p>
-     * The default implementation initiates a {@link EventLoopGroup#shutdownGracefully(long, long, TimeUnit) graceful shutdown}
-     * of the passed {@link EventLoopGroup} instance with no "quiet period" and a timeout of 15 seconds;
-     * then waits uninterruptibly for the shutdown to complete, or the timeout to occur, whichever happens first.
+     * The default implementation initiates a {@link EventLoopGroup#shutdownGracefully() graceful shutdown}
+     * of the passed {@link EventLoopGroup}, then waits uninterruptibly for the shutdown to complete or timeout.
      * <p>
      * Implementation note: if the {@link EventLoopGroup} instance is being shared, or used for other purposes than to
-     * coordinate Netty events for the current cluster, than it should not be shut down here;
+     * coordinate Netty events for the current cluster, then it should not be shut down here;
      * subclasses would have to override this method accordingly to take the appropriate action.
      *
      * @param eventLoopGroup the event loop group used by the cluster being closed
      */
     public void onClusterClose(EventLoopGroup eventLoopGroup) {
-        // shutdownGracefully with default parameters employs a quiet period of 2 seconds
-        // where in pre-Netty4 versions, closing a cluster instance was very quick (milliseconds).
-        // Since we close the channels before shutting down the eventLoopGroup,
-        // it is safe to reduce the quiet period to 0 seconds
-        eventLoopGroup.shutdownGracefully(0, 15, SECONDS).syncUninterruptibly();
+        eventLoopGroup.shutdownGracefully().syncUninterruptibly();
```

Comment:
It does indeed, here are the noticeable ones I found:

* `ClusterInitTest#closeDuringClusterInitTest` went from ~13 seconds to 20 minutes, 
* `ClusterStressTest#clusters_should_not_leak_connections` from 1minute46 seconds to 38 minutes.
* `ControlConnectionTest# should_randomize_contact_points_when_determining_control_connection` from 1.7 sec to 3 min 42 sec

I'll create a NettyOptions in TestUtils called 'nonGracefulShutdownNettyOptions' (or something like that) and apply it to these three classes in particular.

Each Cluster instance we create and close outside of PerClassSingleNodeCluster will add an extra 2 seconds to each test, but I think that is ok.